### PR TITLE
create post page

### PR DESCRIPTION
**Pull request by BuiPhuongThaob**
## Related Tickets:
- [Task #27555](https://edu-redmine.sun-asterisk.vn/issues/27555)

## Notes Impacted Areas:
- N/A.

## Attachments:
title_post
create_post

### CODE REVIEW

**Comment by BuiPhuongThaob** [comment]
ready

File: resources/views/home.blade.php [hunk 1]
```diff
@@ -26,6 +26,8 @@
     <link rel="stylesheet" href="{{ asset('bower_components/review_travel/css_travel/slicknav.css') }}">
 
     <link rel="stylesheet" href="{{ asset('bower_components/review_travel/css_travel/style.css') }}">
+    <link rel="stylesheet" type="text/css" href="{{ asset('bower_components/review_travel/css_travel/post.css') }}">
+<link rel="stylesheet" type="text/css" href="{{ asset('bower_components/select2/dist/css/select2.min.css') }}">
```
Comment (huyenttm-0901) [inline resources/views/home.blade.php]:
```suggestion
    <link rel="stylesheet" type="text/css" href="{{ asset('bower_components/select2/dist/css/select2.min.css') }}">
```

**Comment by BuiPhuongThaob** [comment]
ready

**Review by huyenttm-0901 — COMMENTED** [review]
(merged)

**Pull request merged**